src/main/java/org/humancellatlas/ingest/submission/SubmissionEnvelopeService.java

```diff
@@ -160,6 +170,7 @@ public void handleCommitArchived(SubmissionEnvelope envelope) {
     }
 
     public void handleCommitExported(SubmissionEnvelope envelope) {
+        getProject(envelope).ifPresent(project -> projectService.updateWranglingState(project, WranglingState.SUBMITTED));
```

Comment:
Should this happen when the submission is submitted, at the start of archiving/exporting?
If so this could move to line 134

Reply:
Question answered at standup, this is the correct functionality.

Reply:
Start of exporting happens on `commitExporting`.
For this task we update the project status after exporting is finished, namely when the staging area is ready.
Checking out the [miro board](https://miro.com/app/board/uXjVPcuO51U=/) might shed some light.